src/screens/TopicMemberScreen/index.js

```diff
@@ -186,7 +185,7 @@ const TopicMemberScreen = () => {
         decelerationRate="fast"
         scrollEventThrottle={1}
         style={styles.fragmentContainer}
-        contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
+        contentContainerStyle={[styles.fragmentContentContainer]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding top of `styles.fragmentContentContainer` has been removed. This might affect the layout of your application. If the padding was intentionally removed, then this change is fine. Otherwise, you should add it back.
```diff
- contentContainerStyle={[styles.fragmentContentContainer]}
+ contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->